client/lib/products-values/products-list.ts

```diff
@@ -48,4 +48,16 @@ export const PRODUCTS_LIST = {
 		term: TERM_MONTHLY,
 		bill_period: PLAN_MONTHLY_PERIOD,
 	},
+	[ constants.PRODUCT_WPCOM_SEARCH ]: {
+		product_name: PRODUCT_SHORT_NAMES[ constants.PRODUCT_WPCOM_SEARCH ],
```

Comment:
This is undefined.  You could either add this product to the list in `PRODUCT_SHORT_NAMES` or just define the product name directly here.  Maybe the latter is better since the `PRODUCT_SHORT_NAMES` looks like it's Jetpack-specific and this isn't "really" a Jetpack product (or at least, it's not a product for Jetpack sites).

Reply:
We already use PRODUCT_WPCOM_SEARCH_MONTHLY in the product list, so I opt in for the former: defining it in the PRODUCT_SHORT_NAMES.